pdc/apps/changeset/middleware.py

```diff
@@ -34,9 +34,16 @@ def _may_announce_big_change(self, changeset, request):
                 len(changeset.tmp_changes) < settings.CHANGESET_SIZE_ANNOUNCE):
             return
 
+        author = str(changeset.author)
+        whitelist = getattr(settings, 'BIG_CHANGE_AUTHORS_WHITELIST', set())
+
+        # Skip whitelisted authors.
+        if author in whitelist:
+            return
+
```

Comment:
Not sure if most of people don't want to get the "big change" emails. If so, we will maintain a long list in the future and it might be changed anytime. This is not good.

Maybe we can consider to give an api to do this for each user like obtain token.
Then they can do any change.

Reply:
This change whitelists authors of any big change, not people receiving the e-mails.

Reply:
Yes, I mean if the whitelist is an email list of the people don't want to get e-mails, it should be set somewhere right? Like set in ansible or settings. 
1. If there are many people don't want to get e-mails, we should add them all in config?
2. How we add one more people after upgrade/start the service? Add one string in the BIG_CHANGE_AUTHORS_WHITELIST, not sure it can be used without restart service.

Reply:
@xiangge I don't understand what you mean. We can talk about this tomorrow.

Reply:
@hluk ok